config/crd/bases/airship.airshipit.org_sipclusters.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: (devel)
+    controller-gen.kubebuilder.io/version: v0.2.5
```

Comment:
Changes to this file revert an earlier commit: https://github.com/att-comdev/sip/commit/0f24ea964cb0e43393039ac3813e3018cdad2a36

Reply:
reverted file back to 0f24ea964cb0e43393039ac3813e3018cdad2a36